x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_form.test.tsx

```diff
@@ -28,11 +26,54 @@ import { SearchItems } from '../../../../hooks/use_search_items';
 import { getAggNameConflictToastMessages } from './common';
 import { StepDefineForm } from './step_define_form';
 
+import { MlSharedContext } from '../../../../__mocks__/shared_context';
+import { getMlSharedImports } from '../../../../../shared_imports';
+
 jest.mock('../../../../../shared_imports');
 jest.mock('../../../../app_dependencies');
 
-import { MlSharedContext } from '../../../../__mocks__/shared_context';
-import { getMlSharedImports } from '../../../../../shared_imports';
+const startMock = coreMock.createStart();
+
+const getMockedTimefilter = () => {
```

Comment:
Also, is this different from timefilterServiceMock.createStartContract? Otherwise, this is a pretty useful mock... I wonder if we should move this to a more common place so other tests can also consume it later on?

Reply:
Good catch, it's not different! Reused the supplied mock in [3de83d1](https://github.com/elastic/kibana/pull/149049/commits/3de83d113b1c0330343a6667fb1b749339f1e66f).